test/src/main/scala/scala/test/junit/JunitMultipleSuffixes.scala

```diff
@@ -0,0 +1,25 @@
+package scala.test.junit
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.Assert._
+
+class JunitSuffixE2E {
+
+  @Test
+  def someTest: Unit = {
+  	println("Running E2E")
+  	assertEquals(1, 1)
+  }
+
+}
+class JunitSuffixIT {
+
+  @Test
+  def someTest: Unit = {
+  	println("Running IT")
+  	assertEquals(1, 1)
+  }
+
```

Comment:
Oh and about the danger of the patterns- This is how maven works and is a big need for us to migrate substantial codebases since people won't manage the targets manually. On the other hand they have code supporting the tests side by side with the tests relying on the pattern.
Would you like the default pattern to be "*.class"? I don't like it but can live with it since we'll use our own defaults.